apis/core/v1alpha2/core_types.go

```diff
@@ -125,6 +125,9 @@ type ScopeDefinitionSpec struct {
 	// Reference to the CustomResourceDefinition that defines this scope kind.
 	Reference DefinitionReference `json:"definitionRef"`
 
+	// WorkloadRefPath indicates if/where a scope accepts a workloadRef object
+	WorkloadRefPath string `json:"workloadRefPath,omitempty"`
```

Comment:
For scope, it should be an array of references:
https://github.com/crossplane/oam-kubernetes-runtime/blob/df8e1b1c6beeb38a9a0d3ad16057d157eab9984d/apis/core/v1alpha2/core_scope_types.go#L34

Reply:
That's right. I will fix it.